libretro/libretro_core_options.h

```diff
@@ -113,29 +113,37 @@ struct retro_core_option_definition option_defs_us[] = {
     },
     {
         CORE_NAME "-169screensize",
```

Comment:
Now that I think about it, add a comment that we keep the key for compat.
Also sorry for stretching out this PR a little, just hesitant on stuff with configs ;)

Reply:
No problem. It's understandable. Stuff needs to be done correctly.

Reply:
Is this ok? Or should I change it?
![image](https://user-images.githubusercontent.com/7416381/96594332-dfa3b200-12c0-11eb-8625-ad35dfc473e7.png)

Reply:
Keeping -> Keep, key name -> key.

After the dot add `// TODO: revisit-later` (including the redundant //)

Reply:
Like this?
![image](https://user-images.githubusercontent.com/7416381/96598009-d0266800-12c4-11eb-8e17-97610c6e555e.png)

Anything else?